ml-agents/mlagents/trainers/ppo/policy.py

```diff
@@ -190,14 +190,18 @@ def update(self, mini_batch, num_sequences):
         run_out = self._execute_model(feed_dict, self.update_dict)
         return run_out
 
-    def get_value_estimates(self, brain_info, idx):
+    def get_value_estimates(self, brain_info, idx, done):
```

Comment:
type annotations if it's not too hard to add?

Reply:
Added type annotation and test for this function